lib/features/children/details/widgets/child_details_item.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:givt_app/core/enums/enums.dart';
+import 'package:givt_app/features/auth/cubit/auth_cubit.dart';
+import 'package:givt_app/features/children/details/models/profile_ext.dart';
+import 'package:givt_app/utils/utils.dart';
+import 'package:intl/intl.dart';
+
+class ChildDetailsItem extends StatelessWidget {
+  const ChildDetailsItem({
+    required this.profileDetails,
+    super.key,
+  });
+
+  final ProfileExt profileDetails;
+
+  @override
+  Widget build(BuildContext context) {
+    final user = context.read<AuthCubit>().state.user;
+    final currency = NumberFormat.simpleCurrency(
```

Comment:
Don't we have a Util method for this?

Reply:
good point, I've extended the util class a bit and replaced code above with it in most of the places, thanks